qemu/tests/cfg/remote_image_convert.cfg

```diff
@@ -0,0 +1,81 @@
+# Network storage backends:
+#   iscsi_direct
+#   ceph
+# Cache mode:
+#   only remote image1 -> remote raw image2
+# The following testing scenarios are covered:
+#   remote image1 -> local image -> remote image2
+#   remote image1 -> remote image2
+
+- remote_image_convert:
+    only iscsi_direct ceph
+    virt_test_type = qemu
+    type = remote_convert
+    start_vm = no
+    kill_vm = yes
+    force_create_image = no
+    guest_temp_file = "/var/tmp/convert.tmp"
+    md5sum_bin = "md5sum"
+    convert_source = ${images}
+    convert_target = convert
+    image_name_convert = "images/image_convert"
+    variants:
+        - @default:
+            only to_remote
+            variants:
+                - @default:
+                - cache_mode:
+                    # only for remote->remote raw image converting
+                    only to_remote.to_raw
+                    variants:
+                        - none:
+                            cache_mode = none
+                        - writeback:
+                            cache_mode = writeback
+                        - unsafe:
+                            cache_mode = unsafe
+                        - directsync:
+                            cache_mode = directsync
+                        - writethrough:
+                            cache_mode = writethrough
+        - to_remote:
+            only to_local
+            variants:
+                - to_raw:
+                    image_format_remote = raw
+                - to_qcow2:
+                    image_format_remote = qcow2
+                    ceph:
+                        # qcow2 is supported since 4.1.0 on ceph
+                        required_qemu = [4.1.0,)
+                - to_luks:
+                    image_format_remote = luks
+                    image_secret_remote = convert_remote
+    variants:
+        - to_raw:
+            image_format_convert = raw
+        - to_qcow2:
+            image_format_convert = qcow2
+            ceph:
+                # qcow2 is supported since 4.1.0 on ceph
+                required_qemu = [4.1.0,)
+        - to_luks:
+            image_format_convert = luks
+            image_secret_convert = convert
+    variants:
+        - to_local:
+            # remote -> local(convert)
+            enable_iscsi_convert = no
+            enable_ceph_convert = no
+            image_raw_device_convert = no
+            storage_type_convert = filesystem
+        
+            # local(convert) -> remote(remote)
+            convert_target_remote = remote
```

Comment:
the config name is a little misleading here, better to use `convert_target_convert = remote`?